tests/php/Domain/Services/CreateAccount.php

```diff
@@ -15,6 +15,7 @@
  * that woocommerce_blocks_phase===3, i.e. dev build. Tests will fail
  * with other builds (release feature plugin, woo core package).
  * Related: https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3211
+ * @group TESTS
```

Comment:
I forgot to remove this temporary group, it should be removed. Depending on who merges this pull to `trunk`, let's just make sure this is removed before merge.